### PR TITLE
Add Contributing md file 

### PR DESCRIPTION
**Pull request by willbeaumont**
Issue: #45

## Summary (1-2 sentences)

Added some quick hints on how to get started with our project.

## Details (reason and description of the changes)

This is mainly to help me remember steps for my next PR ;)

### Sources (links that helped to resolve these challenges.) \

Thank you Kent C. Dodds for [How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)!

### CODE REVIEW

**Comment by willbeaumont** [comment]
Followed the merge workflow below, but my header-component changes are showing up in the change log...little confused. Not sure if there was a discussion to add/not add this. No problem closing if the group wants to opt out of this!

```
git checkout main
git pull
git checkout willbeaumont:header-component
git merge main
git push
```

**Review by andrelandgraf — APPROVED** [review]
Thanks for doing this @willbeaumont! I think we can merge it like this and enhance it over time!

Some quick note:

- In Discord or during our tutoring sessions, we can add you as a contributor. Then, you won't need to fork the repo. Instead, you can just clone it directly.
- You will need to create a `.env` file in the project's root. It would probably make sense to add an example `EXAMPLE.ENV` file to the repo with some mock values and add instructions here how to replace them with actual values! (E.g., create a database on [PlanetScale](https://planetscale.com/)).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+Welcome to `social-plan-it`! These are some quick notes on how to start contributing to our open source project. A good place to start is to join our [Discord server](https://discord.gg/tTD7PvwpuX).
```
Comment (tonydangblog) [inline CONTRIBUTING.md:3]: Perhaps we can also mention the Monday meetup here as well!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Setup instructions
+
+1. Fork the repo
+2. Clone your fork
+3. Set your upstream to the project main branch to avoid merge conflicts `git remote add upstream https://github.com/social-plan-it/plan-it-social-web.git`
+4. Create a branch `git checkout -b <name>`
+5. Add your `.env` file (ask the Discord for help on details)
+6. Run `npm install`
+7. Make your changes
+8. Add you changed files with `git add` and `git commit -m "<a stellar commit message>"`
```
Comment (tonydangblog) [inline CONTRIBUTING.md:16]: Small typo here: you -> your

**Review by tonydangblog — APPROVED** [review]
I think it's a great idea to have a contributing doc! It would've been very helpful for when I first joined as well.

I approved and left some comments. I agree with Andre that we can merge as-is and make additional improvements later.

File: CONTRIBUTING.md (hunk 1 above)
Comment (willbeaumont) [reply CONTRIBUTING.md:3]: Great idea!

File: CONTRIBUTING.md (hunk 2 above)
Comment (willbeaumont) [reply CONTRIBUTING.md:16]: 🙏

**Pull request merged**